Gems/PhysX/Code/Tests/PhysXTestUtil.h

```diff
@@ -15,6 +15,9 @@
 
 namespace PhysX
 {
+    // transform for a floor centred at x = 0, y = 0, with top at level z = 0
+    static const AZ::Transform DefaultFloorTransform = AZ::Transform::CreateTranslation(AZ::Vector3::CreateAxisZ(-0.5f));
```

Comment:
nit: Can you please mark this `inline` and not `static` now this is in the header. Thanks!

Reply:
Done